nekoyume/Assets/_Scripts/Game/Game.cs

```diff
@@ -83,6 +83,9 @@ public class Game : MonoSingleton<Game>
         [SerializeField]
         private Prologue prologue;
 
+        [SerializeField]
+        private GameObject debugConsole;
```

Comment:
Could you rename it? It seems a prefab, and it would be better if the name revealed that.

```suggestion
        private GameObject debugConsolePrefab;
```

Reply:
ok~! i will fix that~!